.env

```diff
@@ -1,27 +1,28 @@
-MONGO_URL=mongodb://root:root@localhost:27017/horondi?authSource=admin
-MAIL_USER=horondi@outlook.com
-MAIL_PASS=487Horondi512
-SECRET=4hG7agm$1z5hkiyb4msu5ka84gs5ruy8kfn3u7$8kbd4e5d2j7n6v3jp88gsn58mhd84
-MAIL_HOST=smtp-mail.outlook.com
-MAIL_PORT=587
+MONGO_URL=mongodb+srv://user-read-write1:sfd$42A3a@horondi-yl27g.mongodb.net/horondi-23?retryWrites=true&w=majority
```

Comment:
Remove this file, also `MONGO_URL` in `test` should refer to the locale db

Reply:
Allready remove.